src/puppetlabs/jdbc_util/migration.clj

```diff
@@ -8,15 +8,15 @@
   "Given a user defined database config, transform the config into a db-spec
   appropriate for passing to migratus's migrate function."
   [db-config]
-  (let [user (or (:migration-user db-config)
-                 (:user db-config))
-        password (or (:migration-password db-config)
-                     (:password db-config))]
-    (-> db-config
-        (assoc :user user
-               :password password)
-        (dissoc :migration-user
-                :migration-password))))
+  (let [?user (or (:migration-user db-config)
```

Comment:
heh this is interesting, I had just noticed that (I think it was @aperiodic) RBAC was using the ? prefix to indicate the symbol referred to a promise.